addons/web/static/tests/views/fields/one2many_field_tests.js

```diff
@@ -2966,6 +2966,31 @@ QUnit.module("Fields", (hooks) => {
         // if the changes haven't been saved
     });
 
+    QUnit.test("one2many list: double click on delete record", async function (assert) {
+        serverData.models.partner.records[0].p = [1];
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <form>
+                    <field name="p">
+                        <tree>
+                            <field name="display_name"/>
+                        </tree>
+                    </field>
+                </form>`,
+            resId: 1,
+        });
+
+        setTimeout(() => click(target.querySelector("td.o_list_record_remove")));
+        setTimeout(() => click(target.querySelector("td.o_list_record_remove")));
+        await nextTick();
+        await nextTick();
```

Comment:
```suggestion
        await click(target.querySelector("td.o_list_record_remove"));
        await click(target.querySelector("td.o_list_record_remove"));
```
+ mockRPC to slow down the unlink (grep makeDeferred in this file, you'll find examples)

Reply:
Hi, thanks for the review 🙁 tried to make it work with mockrRPC but I'm not sure if it's possible, maybe if you have another hint 🙏🏽

Reply:
@aab-odoo 🙂

Reply:
I don't see an unlink in rpc, but adding an onchange seems to be working:

```diff
         serverData.models.partner.records[0].p = [1];
+        let def;
         await makeView({
...
-                    <field name="p">
+                    <field name="p" on_change="1">
...
+            async mockRPC(route, args) {
+                if (args.method === "onchange") {
+                    return await Promise.resolve(def).then(() => undefined);
+                }
+            },
         });
 
+        def = makeDeferred();
         setTimeout(() => click(target.querySelector("td.o_list_record_remove")));
         setTimeout(() => click(target.querySelector("td.o_list_record_remove")));
-        await nextTick();
+        def.resolve();
         await nextTick();
```
but there is still one nextTick needed I guess for the read and rerender

Reply:
Thanks for the suggestion, like this seem to work pretty well